notes-with-indexeddb/background.js

```diff
@@ -0,0 +1,14 @@
+/*
+Open a new tab, and load "notes.html" into it.
+*/
+function openNotes() {
```

Comment:
alright - I'm not sure how to do this. the only unique thing about the tab is its url but I don't know how that's generated or if that's the best approach to look for a match.